### PR TITLE
dqsegdb2 update

### PR DESCRIPTION
**Pull request by tdent**
Need a version that supports scitokens

### CODE REVIEW

File: requirements-igwn.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # For LDG service access
-ciecplib[kerberos]>=0.7.0
-dqsegdb2>=1.0.1
+ciecplib[kerberos] >= 0.7.0
+dqsegdb2 >= 1.1.1
```
Comment (duncanmmacleod) [inline requirements-igwn.txt:3]: 1.1.4 is the latest bug fix release for 1.1 that includes a fix for automatically discovering tokens. 

```suggestion
dqsegdb2 >= 1.1.4
```

**Comment by tdent** [comment]
@spxiwh as I have heard Dunk is on holiday, could you approve this please

**Review by spxiwh — APPROVED** [review]
Thanks!

... I know it's convenient, but remember that branches on origin spawn two copies of the CI code, and therefore slow things down. Please delete this after merging!

**Comment by tdent** [comment]
It looks like you already did. I don't have much of an idea how the CI code is set up - do the two copies exist if the PR is not yet merged?

**Comment by spxiwh** [comment]
So there is one copy of all the tests for any pull request *and* there is one copy of all the tests for any commit to origin. So for pull requests from origin we get two copies of both of the tests. A further set of the tests would run after the PR is merged, but that is normal.

.... There are also a few potential security concerns around pull requests from origin.

**Pull request merged**